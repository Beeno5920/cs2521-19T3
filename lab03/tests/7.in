-29 -12 -31 -14 -92 -24 -35 5 6 0 3
