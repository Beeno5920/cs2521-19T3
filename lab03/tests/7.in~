-29 -12 -31 -14 -92 -24 -35 -28 -17 -1 -7 -40 -2 -6 4 0 7
