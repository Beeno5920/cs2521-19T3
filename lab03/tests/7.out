Tree:
       -29
       / \
      /   \
     /     \
    /       \
  -31       -12
  /         / \
-92        /   \
  \      -14    5
  -35    /     / \
       -24    0   6
               \
                3

#nodes = 11,  #leaves = 4
Infix  : -92 -35 -31 -29 -24 -14 -12 0 3 5 6 
Prefix : -29 -31 -92 -35 -12 -14 -24 5 0 3 6 
Postfix: -35 -92 -31 -24 -14 3 0 6 5 -12 -29 
ByLevel: -29 -31 -12 -92 -14 5 -35 -24 0 6 3 
